packages/sources/eth-beacon/src/endpoint/balance.ts

```diff
@@ -114,23 +113,26 @@ const queryInBatches = async (
 ) => {
   const url = `/eth/v1/beacon/states/${params.stateId}/validators`
   const statusList = params.validatorStatus?.join(',')
-  const batchSize = Number(config.adapterSpecificParams?.batchSize) || DEFAULT_BATCH_SIZE
+  const batchSize = Number(config.adapterSpecificParams?.batchSize)
+  const groupSize = Number(config.adapterSpecificParams?.groupSize)
```

Comment:
This looked odd to me as `Number(undefined)` = `NaN`, however it looks like these will actually always be defined, it's just our `Config` type isn't specific enough to be accurate.

This could be improved by adding an `EthBeaconConfig` type to `config.ts`, which more accurately represents the config, which can then be referenced throughout the eth-beacon EA. E.g.
```
export type EthBeaconConfig = Config & {
  adapterSpecificParams: {
    beaconRpcUrl: string,
    executionRpcUrl: string,
    batchSize: number,
    groupSize: number,
    chainId: number
  }
}
```

Feel free to take it or leave it, not required but would better enforce our types and avoid needing to have unnecessary undefined handling!

Reply:
I'll take it! The plan is to eventually move these adapters to v3 though which provider stricter typing for this but think it's worth making this change in the meantime